apm-sniffer/optional-plugins/optional-spring-plugins/optional-spring-cloud/gateway-2.1.x-plugin/pom.xml

```diff
@@ -34,6 +34,7 @@
 
 <properties>
     <spring-cloud-starter-gateway.version>2.1.1.RELEASE</spring-cloud-starter-gateway.version>
+    <compiler.version>1.8</compiler.version>
```

Comment:
Are you using jdk8 style api? Why do you change this?

Reply:
yes, i use Mono.doFinally()

Reply:
sorry i have some changes please look at the last commit